src/eggroll-in-a-bowl.md

```diff
@@ -0,0 +1,34 @@
+# Egg Roll in a Bowl
+
+Egg rolls, but, you know, in a bowl.
+
+- ⏲️ Prep time: 10 min
+- 🍳Cook time: 15 min
+- 🍽️ Serving 22
```

Comment:
Could you elaborate on what `22` means?

Reply:
No it is 2 servings. I was having issues with markdown rendering in vim. I think it has something to do with emojis. It looked right when I pushed it. Apologies.